### PR TITLE
add surgical medkit to cargo

### PR DESCRIPTION
**Pull request by sippykot**
## About The Pull Request
Adds surgical medkit to cargo goodies

## Why It's Good For The Game
Items you can only get at roundstart are bad, ackshually

## Proof Of Testing
i ran it, it works, it's a really simple change

## Changelog

:cl:
add: you can buy the surgical medkit in cargo now

/:cl:

### CODE REVIEW

**Comment by sippykot** [comment]
why the hell did it fail the birdshot check? how? could it be that that map sucks ass?

**Comment by lessthnthree** [comment]
Maps will fail randomly because of storyteller, don't worry about it

**Comment by projectkepler-ru** [comment]
> ## Why It's Good For The Game
> Items you can only get at roundstart are bad, ackshually

Not necessarily, item that can only be obtained by joining round start & late joining aren't all that bad.

File: modular_zubbers/code/modules/cargo/packs/goodies.dm [hunk 1]
```diff
@@ -74,3 +74,9 @@
 
 /datum/supply_pack/goody/thermal_single
 	special = FALSE
+
+/datum/supply_pack/goody/medkit_surgery
+	name = "Spare Surgical Medkit"
+	desc = "A spare high capacity medkit containing a set of surgical tools and basic supplies."
+	cost = PAYCHECK_CREW * 10
```
Comment (lessthnthree) [inline modular_zubbers/code/modules/cargo/packs/goodies.dm:81]:
```suggestion
	cost = PAYCHECK_CREW * 15
```

Comment (lessthnthree) [reply modular_zubbers/code/modules/cargo/packs/goodies.dm:81]: This box has 12 slots and fits in a backpack. It is a premium item.

**Review by lessthnthree — CHANGES_REQUESTED** [review]

**Comment by lessthnthree** [comment]
Actually, why not just place it in the nanomed? Saves everyone time.

File: modular_zubbers/code/modules/cargo/packs/goodies.dm (hunk 1 above)
Comment (sippykot) [reply modular_zubbers/code/modules/cargo/packs/goodies.dm:81]: yeah i guess it could be more expensive - can lower the price later if people complain

**Comment by sippykot** [comment]
> Actually, why not just place it in the nanomed? Saves everyone time.

it's an option - could also do both, i think the coroner medkit is in the vendor and in a cargo crate

**Comment by sippykot** [comment]
should probably be cheaper in the vending machine since you dont pay for the shipping... not sure how much cheaper tho

**Comment by lessthnthree** [comment]
I think 750 is cheap given the surgery tray is 1200 and can't be put in a backpack.

**Comment by lessthnthree** [comment]
> > Actually, why not just place it in the nanomed? Saves everyone time.
> 
> it's an option - could also do both, i think the coroner medkit is in the vendor and in a cargo crate

Yes, and the autopsy scanner is 100 in the machine and 500 from cargo, for a comparison.

**Review by lessthnthree — APPROVED** [review]

**Pull request merged**